pkg/system_api.go

```diff
@@ -493,6 +508,20 @@ func (components Components) Summarize() v1.Summary {
 	return s
 }
 
+func (component Component) GetStatus() string {
+	if component.Summary.Healthy > 0 && component.Summary.Unhealthy > 0 {
+		return "warning"
```

Comment:
Should we make these as constants ? Saw this being used at other places too

Reply:
Yeah, I think so too... we should be creating a status type and have it only have these values